modules/rosa-hcp-vpc-manual.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PREFERENCE
+[id="rosa-hcp-vpc-manual_{context}"]
+=== Creating a Virtual Private Cloud (VPC) manually
```

Comment:
```suggestion
= Creating a Virtual Private Cloud (VPC) manually
```

Reply:
IIRC, I think you also should move the (VPC) to the first appearance in the text rather than the module title